db/migrate/20190806195206_user_model.rb

```diff
@@ -0,0 +1,11 @@
+class UserModel < ActiveRecord::Migration[5.2]
+  def change
+    create_table :user do |t|
+      t.string :name
```

Comment:
The Odin Project [(this one)](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations) specified that in the user's login we used a name or ID instead of an email, that's why we used that instead of emails.